src/Util/LowercaseMethodHelper.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
Is this file useful?

Reply:
Ouch, sorry – leftover from some other approach.